### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by kanika-saini**
What does this PR do?
Add resource

Description
Computer Networks Course (Pearson)

How do we know it's really free?
I took the course myself

###For course lists, is it a course

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -802,6 +803,10 @@
 
 > :information_source: See also &#8230; [C#](#csharp)
 
+### Networking
+
+* [Computer Networks | Tanenbaum](https://media.pearsoncmg.com/ph/streaming/esm/tanenbaum5e_videonotes/tanenbaum_videoNotes.html) - Pearson)
```
Comment (eshellman) [inline courses/free-courses-en.md:808]: Don't invent titles! Use the title given by the resource. 
Please add instructor names where appropriate.
Escape pipe characters

Comment (kanika-saini) [reply courses/free-courses-en.md:808]: I am sorry eric , i don't quite understand what you meant by inventing titles. There was no section for networking resources so i added one. And the title of the course is the same as on Pearson media website. 
Here is the attached screenshot.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -802,6 +803,10 @@
 
 > :information_source: See also &#8230; [C#](#csharp)
 
+### Networking
+
+* [Computer Networks 5e](https://media.pearsoncmg.com/ph/streaming/esm/tanenbaum5e_videonotes/tanenbaum_videoNotes.html) - Tanenbaum, Wetherall ( Pearson)
```
Comment (davorpa) [inline courses/free-courses-en.md:808]: Complete instructors

```suggestion
* [Computer Networks 5e](https://media.pearsoncmg.com/ph/streaming/esm/tanenbaum5e_videonotes/tanenbaum_videoNotes.html) - Andrew Tanenbaum, David Wetherall (Pearson)
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -57,6 +57,7 @@
 * [Matlab](#matlab)
 * [Misc](#misc)
 * [.NET](#net)
+* [Networking](#Networking)
```
Comment (davorpa) [inline courses/free-courses-en.md:60]: fragments ids goes in lowercase

```suggestion
* [Networking](#networking)
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**